activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/src/main/java/org/activiti/services/connectors/model/IntegrationEvent.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class IntegrationEvent {
```

Comment:
@erdemedeiros as suggested by mario, we should rename this to IntegrationRequestEvent. I think that it is much clear.

Reply:
I agree.

Reply:
BTW this can be applied to the persisted model in Activiti/Activiti as well. `IntegrationContextEntity` can be renamed to `IntegrationRequestEntity`. What do you think?

Reply:
@erdemedeiros the problem that I have with that is that in the core engine level we need to track both IntegrationRequest and IntegrationResult in that entity. I was thinking more about adding a status (Pending) when the integrationRequest is sent and completed when the integrationResult arrived. What do you think?

Reply:
It makes sense. Or even keep it as it's (without status) and delete the entry once the integrationResult arrives. What do you think? Should we keep a history on that? I'm afraid that this table will grow really fast.

Reply:
@erdemedeiros removing it will make sense.. as soon as we have the audit events.. so we know what happened.